third_party/move/move-model/src/builder/exp_builder.rs

```diff
@@ -136,6 +136,52 @@ pub(crate) enum OldExpStatus {
     InsideOld,
 }
 
+#[derive(Debug, PartialEq, Eq, Copy, Clone)]
+pub enum ModelCallKind {
```

Comment:
This relates to the `_` notation.  It really would be confusing if `_ + 3` turns into a function.

Maybe I should try to remove the `_` changes.  It may be more trouble than it's worth.